libcontainer/integration/exec_test.go

```diff
@@ -1921,6 +1921,9 @@ func TestFdLeaks(t *testing.T) {
 }
 
 func TestFdLeaksSystemd(t *testing.T) {
+	if !systemd.IsRunningSystemd() {
+		t.Skip("Systemd is unsupported")
```

Comment:
"test requires systemd" might be less confusing.

Reply:
@AkihiroSuda this is what we already have in other cases:

```
--
libcontainer/integration/exec_test.go:  if !systemd.IsRunningSystemd() {
libcontainer/integration/exec_test.go-          t.Skip("Systemd is unsupported")
--
libcontainer/integration/exec_test.go:  if !systemd.IsRunningSystemd() {
libcontainer/integration/exec_test.go-          t.Skip("Systemd is unsupported")
--
libcontainer/integration/exec_test.go:  if !systemd.IsRunningSystemd() {
libcontainer/integration/exec_test.go-          t.Skip("Systemd is unsupported")
--
libcontainer/integration/exec_test.go:  if !systemd.IsRunningSystemd() {
libcontainer/integration/exec_test.go-          t.Skip("Systemd is unsupported")
--
libcontainer/integration/exec_test.go:  if !systemd.IsRunningSystemd() {
libcontainer/integration/exec_test.go-          t.Skip("Systemd is unsupported")
--
```

Reply:
Yeah I copied this from the other integration functions. If we really want to change it we can but we should change it for all of them in one commit.